pages/index.tsx

```diff
@@ -5,8 +5,9 @@ import Heading from "@components/text/Heading";
 import InternalButton from "@components/button/InternalButton";
 import StepperTimeline from "@components/figure/Timeline";
 import { InformationBanner } from "@components/figure/Banner";
+import { ReactNode } from "react";
 
-const Section: React.FC<{ isBlue?: boolean }> = ({ children, isBlue = false }) => (
+const Section: React.FC<{ children?: any, isBlue?: boolean }> = ({ children, isBlue = false }) => (
```

Comment:
Minor TypeScript suggestion

```suggestion
const Section: React.FC<{ children?: React.ReactNode, isBlue?: boolean }> = ({ children, isBlue = false }) => (
```